style.css

```diff
@@ -467,3 +497,241 @@ button:active {
 .form-cantact button {
   margin-left: -190px;
 }
+
+@media (min-width: 489px) and (max-width: 767px) {
+  .toolbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .mylogo {
+    margin-left: 150px;
+  }
+
+  .nav {
+    display: flex;
+    justify-content: flex-end;
+    margin-left: 390px;
+  }
+
+  .nav ul {
+    display: flex;
+  }
+
+  .nav-item {
+    margin-left: 444px;
+  }
+
+  .lang li {
+    display: flex;
+    flex-direction: column;
+    width: 122px;
+    height: 120px;
+  }
+
+  .nav ul li {
+    margin: 20px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .first-page {
+    background-image: url(./image/dec-intro.png);
+    background-position: left;
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 100%;
+    min-height: 100vh;
+    padding: 22px;
+  }
+
+  .headl1 {
+    margin: 5em 18em 0;
+    width: 600px;
+  }
+
+  .card-image img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .mob {
+    display: none;
+  }
+
+  .dec {
+    display: block;
+  }
+
+  .card-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    background: #fff;
+    border-radius: 24px;
+    padding: 1rem;
+    height: 486px;
+    width: 1156px;
+    margin: 0 auto;
+    gap: 3.5em;
+  }
+
+  .two,
+  .four {
+    flex-direction: row-reverse;
+  }
+
+  .about-page {
+    display: flex;
+    flex-direction: row;
+    padding: 2.5em;
+  }
+
+  .about-me-left {
+    width: 40%;
+  }
+
+  .about-me-right {
+    width: 60%;
+  }
+
+  .lang {
+    display: flex;
+  }
+
+  .frame-arro {
+    display: flex;
+    justify-content: flex-end;
+    position: relative;
+    left: -46px;
+  }
+
+  .form-cantact {
+    background-image: url(./image/dec-form.png);
+  }
+}
+
+@media (min-width: 768px) {
+  .toolbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .mylogo {
+    margin-left: 150px;
+  }
+
+  .nav {
+    display: flex;
+    justify-content: flex-end;
+    margin-left: 390px;
+  }
+
+  .nav ul {
+    display: flex;
+  }
+
+  .nav-item {
+    margin-left: 444px;
+  }
+
+  .lang li {
+    display: flex;
+    flex-direction: column;
+    width: 122px;
+    height: 120px;
+  }
+
+  .nav ul li {
+    margin: 10px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .first-page {
+    background-image: url(./image/dec-intro.png);
+    background-position: left;
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 100%;
+    min-height: 100vh;
+    padding: 22px;
+  }
+
+  .headl1 {
+    margin: 5em 18em 0;
+    width: 600px;
+  }
+
+  .card-image img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .mob {
+    display: none;
+  }
+
+  .dec {
+    display: block;
+  }
+
+  .card-container {
+    display: flex;
```

Comment:
-  [OPTIONAL] :thinking: In my opinion, it would be better to make your whole project responsive by using percentages instead of px for width.
[Screencast from 01-20-2023 10:09:38 PM.webm](https://user-images.githubusercontent.com/97834160/213796997-d8b70a58-820b-458d-8e78-00ae07fa1c55.webm)